htpy/__init__.py

```diff
@@ -10,6 +10,8 @@
 from markupsafe import Markup as _Markup
 from markupsafe import escape as _escape
 
+from .html2htpy import *
+
```

Comment:
Is this import needed? I think we can just remove it? And then import like `from htpy.html2htpy import html2htpy`? I think mypy/type checkers should pick up that properly! I think very few people will import the html2htpy function, this feature will mostly be used from the cli anyways. But it is nice to have a python API for it too.

Reply:
In this case, it is indeed not needed! I will remove it.